edwin/client/static/js/components/Timeline.js

```diff
@@ -24,22 +27,34 @@ import Icon from './Icon.js';
  */
 export default class Timeline extends ControllerComponent {
   get stores() {
-    return [BugStore, UserStore];
+    return [BugStore, UserStore, IndicatorStore];
   }
 
   loadData() {
     let teamSlug = this.props.params.team;
     let bugQuery = {comment_tag: `edwin-${teamSlug}`};
 
+    IndicatorActions.updateState(LoadingStates.USER_RESTORE);
```

Comment:
Hrm. I think I'd rather see these updateState calls happen in thet action creators (like `TimelineActions.loadTeams`) so they are more centralized when we start having other top-level components.